src/SFA.DAS.AssessorService.Database/StoredProcedures/AssessmentsSummaryUpdate.sql

```diff
@@ -0,0 +1,57 @@
+﻿-- Data for GET /api/ao/assessments?standard={IfateReferenceNumber}&ukprn={ukprn}
+--
+CREATE PROCEDURE [dbo].[AssessmentsSummaryUpdate]
```

Comment:
This was copied from the other one which used a try catch and transaction for a good reason - this version does not need that so let the C# handle it

Reply:
Done